views/components/Graphs/GraphTable.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import {
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Table,
+  Paper,
+  Toolbar,
+  Typography,
+} from '@material-ui/core'
+
+const GraphTable = ({ graph }) => {
+  return (
+    <Paper>
+      <Toolbar>
+        <Typography variant="h6" color="inherit">
+          {graph?.title.toUpperCase()} TABLE
+        </Typography>
+      </Toolbar>
```

Comment:
I think we can remove this because we're already on the page so have this context.

Reply:
https://github.com/AMP-SCZ/dpdash/pull/175/commits/833d8ffc41e3d610e2ef099e378fb27d1d223c98